README.md

```diff
@@ -26,22 +26,18 @@ Sentry SDK for Java and Android
 
 
 
-# Java SDK 3.0 Docs
+# Useful links and docs
 
-The Java SDK documentation [can be found on docs.sentry.io](https://docs.sentry.io/platforms/java/).
```

Comment:
`m`: Shouldn't we keep a link to the SKD docs?

Reply:
> removed dup. links (which is already under Resources)